docker/docker_official_images/3.8.0/jvm/Dockerfile

```diff
@@ -92,4 +87,4 @@ USER appuser
 
 VOLUME ["/etc/kafka/secrets", "/var/lib/kafka/data", "/mnt/shared/config"]
 
-CMD ["/etc/kafka/docker/run"]
+CMD ["/etc/kafka/docker/run"]
```

Comment:
I'm definitely still struggling to understand the necessity of so much Docker-image-custom behavior -- what problem is being solved here that's _unique_ to running Kafka inside a container?  I can't think of very many things that would be useful for containers that wouldn't _also_ make the life of users in things like high-constrained systemd units better, such that any logic here should probably live in proper upstream scripts / code instead.